### PR TITLE
Get greenkeeper-lockfile working for Jenkins pipelines.

### PR DESCRIPTION
**Pull request by chrisleck**
Fix for #68.

### CODE REVIEW

File: ci-services/jenkins.js [hunk 1]
```diff
@@ -3,10 +3,18 @@ const env = process.env
...
+// pipeline vs. older job types.
+const branchName = env.CHANGE_BRANCH || env.GIT_BRANCH
+const gitUrl = env.CHANGE_URL || env.GIT_URL
+
+// Build number is available in Jenkins pipeline builds; for others, defer to asking GIT.
+const buildNumber = env.BUILD_NUMBER || gitHelpers.getNumberOfCommitsOnBranch(branchName)
+
 module.exports = {
-  gitUrl: env.GIT_URL,
-  branchName: _.drop(_.split(env.GIT_BRANCH, '/')).join('/'),
```
Comment (janl) [inline ci-services/jenkins.js]: The `branchName` mangling code seems to not have survived this. Any particular reason?

File: ci-services/jenkins.js [hunk 2]
```diff
@@ -3,10 +3,18 @@ const env = process.env
 const _ = require('lodash')
```
Comment (janl) [inline ci-services/jenkins.js]: Leftover lodash require, either remove, or address my other comment for old line 8.

**Comment by kaylee42** [comment]
I pulled down these changes and addressed code review - put in a new PR at #148 as couldn't push to this branch

**Comment by hulkoba** [comment]
closing because #148 is merged

**Pull request closed without merging**